resources/css/app.css

```
@@ -111,20 +113,21 @@
     --sidebar-accent-foreground: oklch(0.205 0 0);
     --sidebar-border: oklch(0.922 0 0);
     --sidebar-ring: oklch(0.87 0 0);
+    --footer-background: oklch(0.21 0.034 264.665);
```

Should be named just footer and should have foreground color for text.
```suggestion
    --footer: oklch(0.21 0.034 264.665);
    --footer-foreground: HSL(180°, 0%, 90%)
```